bigtable/tests/unit/test_cluster.py

```diff
@@ -49,38 +82,117 @@ def _make_client(self, *args, **kwargs):
         return self._get_target_client_class()(*args, **kwargs)
 
     def test_constructor_defaults(self):
-        from google.cloud.bigtable.cluster import DEFAULT_SERVE_NODES
-
         client = _Client(self.PROJECT)
         instance = _Instance(self.INSTANCE_ID, client)
 
-        cluster = self._make_one(self.CLUSTER_ID, instance, self.LOCATION_ID)
+        cluster = self._make_one(self.CLUSTER_ID, instance)
         self.assertEqual(cluster.cluster_id, self.CLUSTER_ID)
         self.assertIs(cluster._instance, instance)
-        self.assertEqual(cluster.serve_nodes, DEFAULT_SERVE_NODES)
+        self.assertIsNone(cluster.location_id)
+        self.assertIsNone(cluster._state)
```

Comment:
Test the property here, rather than the attribute (which will fix the coverage issue, as well).

Reply:
Yes
Changed it in system test, but forgot to change in unit test.
That is also the reason for the drop test coverage.

Done